docs/mofed-container-env-vars.md

```diff
@@ -23,6 +23,7 @@ In most deployments its expected to not require setting those.
 | UNLOAD_STORAGE_MODULES |N|`"false"`| unload host storage modules prior to loading mofed modules  |
 | ENABLE_NFSRDMA |N|`"false"`| enable loading of nfs relates storage modules from mofed container|
 | RESTORE_DRIVER_ON_POD_TERMINATION |N|`"true"`| restore host drivers when container is gracefully stopped |
+| USE_DRIVERS_INVENTORY |N|`"true"`| use a persistent directory to store drivers' build artifacts to avoid recompilation between runs |
```

Comment:
So, this is an Operator variable, not a MODED container one.
I don't think we have a doc for Operator variables, maybe we should

Reply:
`USE_DTK` env var can be controlled by a helm chart and is documented as part of it. Do we want to do the same with this feature?

Reply:
Yes, I think we could align the same as `USE_DTK`